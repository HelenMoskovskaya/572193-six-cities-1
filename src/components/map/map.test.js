import React from "react";
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const mockOffer = [
  {
    coords: [52.3909553943508, 4.85309666406198],
    title: `Test`,
    type: `Test`,
    image: `img/test-01.jpg`,
    price: 30,
    rating: 39,
    isPremium: false,
  },
  {
    coords: [52.369553943508, 4.85309666406198],
    title: `Test`,
    type: `Test`,
    image: `img/test-02.jpg`,
    price: 320,
    rating: 89,
    isPremium: false,
  },
  {
    coords: [52.3909553943508, 4.929309666406198],
    title: `Test`,
    type: `Room`,
    image: `img/test-03.jpg`,
    price: 620,
    rating: 100,
    isPremium: true,
  }
];

it(`Map correcrly renders`, () => {
  const tree = renderer
    .create(<Map
      offers={mockOffer}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
