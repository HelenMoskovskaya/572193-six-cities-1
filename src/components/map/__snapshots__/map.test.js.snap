// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map correcrly renders 1`] = `
<div
  id="map"
  style={
    Object {
      "height": 800,
      "margin": "25px 0 25px 0",
    }
  }
/>
`;
